packages/url-loader/src/constants/parameters.ts

```diff
@@ -126,6 +128,17 @@ export const flags = {
     }))
 }
 
+/** Format */
+
+export const format = {
+  qualifier: 'f',
+  // @TODO: enum
```

Comment:
suggestion: maybe we could add this enum as it was done above?

```ts
export const aspectRatioModesEnum = z.enum([
  'vflip',
  'hflip',
  'ignore',
  'auto_right',
  'auto_left',
]);
```

Reply:
https://github.com/colbyfayock/cloudinary-util/issues/142